src/os_win/os_fs.c

```diff
@@ -497,13 +484,6 @@ __win_open_file(WT_FILE_SYSTEM *file_system, WT_SESSION *wt_session, const char
     } else
         dwCreationDisposition = OPEN_EXISTING;
 
-    /* Direct I/O. */
-    if (LF_ISSET(WT_FS_OPEN_DIRECTIO)) {
-        f |= FILE_FLAG_NO_BUFFERING;
-        win_fh->direct_io = true;
-    }
-
-    /* FILE_FLAG_WRITE_THROUGH does not require aligned buffers */
     if (FLD_ISSET(conn->write_through, file_type))
```

Comment:
Earlier it said write through is part of the Windows equivalent of Direct I/O. Should all the write through code be deleted? If not, comment old line 506 should not be deleted.

Reply:
Hi Tod, the write through code is useful in other areas outside of configuring it for direct_io, so we made the decision to keep write through. CC: @korteland

Reply:
Yep - there are durability ramifications for Windows when not using write-through.